layouts/contact/single.html

```diff
@@ -0,0 +1,34 @@
+{{ partial "general-title" . }}
+
+{{ partial "header" . }}
+    {{ partial "navbar" . }}
+    <!-- Main -->
+    <div id="main">
+        <article class="post">
+            {{ $.Scratch.Set "h1" true }}
+            {{ .Render "header" }}
+            <div id="content">
+                {{ .Content }}
+                <div class="contact-container">
+                <form id="contact" action="//formspree.io/{{ .Site.Social.email }}" method="post" center width="300px">
+                  <h3>Contact Me</h3>
+                  <h4>Replies Typically Within 24 Hours.</h4>
+                  <fieldset>
```

Comment:
I had issues customizing the fields with a Param.* but I am doubling back once I have some time.